rqd/rqd/cuerqd.py

```diff
@@ -18,20 +18,20 @@
 """SYNOPSIS
      cuerqd [hostname] [OPTIONS]
       [hostname]            => RQD hostname (defaults to localhost)
-      -s                    => Print rqd status
-      -v                    => Print rqd version
-      --lp <cores>          => Lock Cores
-      --ulp <cores>         => Unlock Cores
-      --lh                  => Lock Host (All Cores)
-      --ulh                 => Unlock Host (All Cores)
-      --nimbyoff            => Turn on Nimby
-      --nimbyon             => Turn off Nimby
+      -s or --status        => Print rqd status
+      -v or --version       => Print rqd version
+      --lock <cores>        => Lock Cores
+      --unlock <cores>      => Unlock Cores
+      --lock all            => Lock Host (All Cores)
+      --unlock all          => Unlock Host (All Cores)
+      --nimby on            => Turn on Nimby
+      --nimby off           => Turn off Nimby
       --exit                => Lock host, wait until machine is idle and then Shutdown RQD *
-      --exit_now            => KILL ALL running frames and Shutdown RQD
+      --exit now            => KILL ALL running frames and Shutdown RQD
       --restart             => Lock host, wait until machine is idle and then Restart RQD *
-      --restart_now         => KILL ALL running frames and Restart RQD
+      --restart now         => KILL ALL running frames and Restart RQD
```

Comment:
Could we please update 'Restart' to 'restart' (lowercase)

Reply:
Updated all requested changes.

Reply:
@sharifsalah Please review.

Reply:
Thanks! That looks good to me.

Reply:
Please see #667.